corehq/apps/reports/static/reports/js/filters/case_list_explorer.js

```diff
@@ -38,9 +38,13 @@ hqDefine("reports/js/filters/case_list_explorer", ['jquery', 'underscore', 'knoc
         self.label = ko.observable(label).trimmed();
 
         self._value = function () {
-            return _.find($parent.allSuggestions, function (prop) {
+            const valueList = _.filter($parent.allSuggestions, function (prop) {
                 return prop.name === self.name();
             });
+            if (valueList.length === 1) {
```

Comment:
@zandre-eng Will there ever be a case where this will be greater than 1?

Reply:
There will, specifically when more than one case type has the same case property name.

Reply:
I'm not totally clear on what this _value method should do. Is it the case that it's simply trying to find whether a unique property (name) exists?

Reply:
This method gets called whenever `name` changes (in `self.name.subscribe()`). Essentially, whenever the user selects a case property from the suggested case property dropdown, `_value()` will be called to get the label for that case property.

Reply:
Ah, that makes sense. Thanks for clarifying.